release/serve_tests/workloads/deployment_graph_long_chain.py

```diff
@@ -0,0 +1,181 @@
+@click.option("--init-delay-secs", type=int, default=0)
+@click.option("--compute-delay-secs", type=int, default=0)
+@click.option(
+    "--num-requests-per-client",
+    type=int,
+    default=DEFAULT_NUM_REQUESTS_PER_CLIENT,
+)
+@click.option("--num-clients", type=int, default=DEFAULT_NUM_CLIENTS)
+@click.option(
+    "--throughput-trial-duration-secs",
+    type=int,
+    default=DEFAULT_THROUGHPUT_TRIAL_DURATION_SECS,
+)
+@click.option("--local-test", type=bool, default=True)
+def main(
+    chain_length: Optional[int],
+    init_delay_secs: Optional[int],
+    compute_delay_secs: Optional[int],
+    num_requests_per_client: Optional[int],
+    num_clients: Optional[int],
+    throughput_trial_duration_secs: Optional[int],
+    local_test: Optional[bool],
+):
+    if local_test:
+        setup_local_single_node_cluster(1, num_cpu_per_node=8)
+    else:
+        setup_anyscale_cluster()
+
+    serve_dag = test_long_chain_deployment_graph(
+        chain_length,
+        init_delay_secs=init_delay_secs,
+        compute_delay_secs=compute_delay_secs,
+    )
+    dag_handle = serve.run(serve_dag)
+
+    # 1 + 2 + 3 + 4 + ... + chain_length
+    expected = ((1 + chain_length) * chain_length) / 2
+    assert ray.get(dag_handle.predict.remote(0)) == expected
+    loop = asyncio.get_event_loop()
+
+    throughput_mean_tps, throughput_std_tps = loop.run_until_complete(
+        benchmark_throughput_tps(
+            dag_handle,
+            expected,
+            duration_secs=throughput_trial_duration_secs,
+            num_clients=num_clients,
+        )
+    )
+    latency_mean_ms, latency_std_ms = loop.run_until_complete(
+        benchmark_latency_ms(
+            dag_handle,
+            expected,
+            num_requests=num_requests_per_client,
+            num_clients=num_clients,
+        )
+    )
+    print(f"chain_length: {chain_length}, num_clients: {num_clients}")
+    print(f"latency_mean_ms: {latency_mean_ms}, " f"latency_std_ms: {latency_std_ms}")
+    print(
+        f"throughput_mean_tps: {throughput_mean_tps}, "
+        f"throughput_std_tps: {throughput_std_tps}"
+    )
+
+    results = {
+        "chain_length": chain_length,
+        "init_delay_secs": init_delay_secs,
+        "compute_delay_secs": compute_delay_secs,
+        "local_test": local_test,
+    }
+    results["perf_metrics"] = [
+        {
+            "perf_metric_name": "throughput_mean_tps",
+            "perf_metric_value": throughput_mean_tps,
+            "perf_metric_type": "THROUGHPUT",
+        },
+        {
+            "perf_metric_name": "throughput_std_tps",
+            "perf_metric_value": throughput_std_tps,
+            "perf_metric_type": "THROUGHPUT",
+        },
+        {
+            "perf_metric_name": "latency_mean_ms",
+            "perf_metric_value": latency_mean_ms,
+            "perf_metric_type": "LATENCY",
+        },
+        {
+            "perf_metric_name": "requests_std_ms",
```

Comment:
```suggestion
            "perf_metric_name": "latency_std_ms",
```
unless I'm missing something?